ecommerce-ui/src/ShoppingCart.js

```diff
@@ -0,0 +1,28 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import VacationRental from './VacationRental';
+
+class ShoppingCart extends Component {
+
+  render() {
+	  
+	const rentals = this.props.rentals.map((vacationRental, index) => {
```

Comment:
My only big feedback would be to clean up spacing next time. Indenting functions/JSX properly helps make code easier to read.